app/uk/gov/hmrc/eusubsidycompliancefrontend/services/EscService.scala

```diff
@@ -55,18 +55,19 @@ class EscService @Inject() (escConnector: EscConnector)(implicit ec: ExecutionCo
       case Left(_) => Right(None)
       case Right(value) =>
         value.status match {
-          case NOT_FOUND => Right(None)
           case NOT_ACCEPTABLE =>
```

Comment:
Bit confused about this. It's still appropiate for the backend to return a 404 if the undertaking is not found. Is the backend not returning a 404 anymore?

Reply:
The EIS system is like that, it gives OK response for everything and hence in our BE we compare the Error code and decide the status. Otherwise everything is OK